lib/experimental/fonts/font-library/class-wp-font-collection.php

```diff
@@ -0,0 +1,90 @@
+<?php
+/**
+ * Font Collection class.
+ *
+ * This file contains the Font Collection class definition.
+ *
+ * @package    WordPress
+ * @subpackage Font Library
+ * @since      6.4.0
+ */
+
+if ( class_exists( 'WP_Font_Collection' ) ) {
+	return;
+}
+
+/**
+ * Font Collection class.
+ *
+ * @since 6.4.0
+ */
+class WP_Font_Collection {
+
+	/**
+	 * Font collection configuration.
+	 *
+	 * @since 6.4.0
+	 *
+	 * @var array
+	 */
+	private $config;
+
+	/**
+	 * WP_Font_Collection constructor.
+	 *
+	 * @since 6.4.0
+	 *
+	 * @param array $config Font collection config options.
+	 * @throws Exception If the required parameters are missing.
+	 */
+	public function __construct( $config ) {
+		if ( empty( $config ) ) {
```

Comment:
```suggestion
		if ( empty( $config ) || ! is_array( $config ) ) {
```

Reply:
I'm not sure about this change. $config should be an array, not a string, right?

Reply:
Hahah sorry! I misread `is_array` for `is_string` :sweat_smile: the change looks good